### ISSUE TITLE
Auto-instantiate forms when type-hinted in route parameters

### ISSUE THREAD

**Issue by Tina-otoge** [asso-msn/msn-web#1]
Just like FastAPI schemas.

Use https://github.com/OrganIA/organia-server/blob/master/app/application.py as reference.

**Comment by Tina-otoge** [comment]
Old code:

```py
@app.get("/register/")
def register():
    form = RegisterForm()
    return app.render("pages/register", form=form)


@app.post("/register/")
def register_post():
    form = RegisterForm()
    if not form.validate_on_submit():
        return app.redirect("register")
    return "Registered"
```

New code:

```py

@app.get("/register/")
def register(form: RegisterForm):
    return app.render("pages/register", form=form)


@app.post("/register/")
def register_post(form: RegisterForm):
    if not form.validate_on_submit():
        return app.redirect("register")
    return "Registered"
```

**Comment by Tina-otoge** [comment]
Closed in #19
